css/style.css

```diff
@@ -0,0 +1,199 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  color: #333;
+  font-family: Helvetica, Arial, Verdana, sans-serif;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  width: 100%;
+  height: 100%;
+}
+
+ul li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1,
+h2 {
+  font-family: 'Lato', Helvetica, sans-serif;
+}
+
+h1 {
+  font-size: 4rem;
+  font-weight: 100;
+  margin: 30px 0 20px 0;
+}
+
+h2 {
+  font-size: 2rem;
+  font-weight: 300;
+}
+
+footer a {
+  color: #39aee9;
+}
+
+nav li a {
+  border-width: 0 2px 0 1px;
+  height: 36px;
+  color: #fff;
+}
+
+nav img {
+  height: 24px;
+  width: 24px;
+}
+
+main img {
+  position: absolute;
+}
+
+nav ul li {
+  float: left;
+}
+
+#footer-links {
+  float: right;
+  font-size: 11px;
+  margin-top: -20px;
+  margin-right: 10px;
+}
+
+.line {
+  border-left: 1px solid #ccc;
+}
+
+#search {
+  margin-left: 1px;
+  padding: 5px;
+  z-index: 2;
+  color: white;
+  position: absolute;
+}
+
+.heading {
+  margin-bottom: 25px;
+}
+
+footer p {
+  clear: both;
+  width: 49%;
+  font-size: 11px;
+  padding-top: 1.5em;
+  text-align: left;
+  margin-left: 200px;
+  color: #6e6e6e;
+}
+
+.heading a {
+  color: white !important;
```

Comment:
#### `REQUIRED`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. 

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the !important rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.